src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -220,11 +220,11 @@ void ICACHE_RAM_ATTR SX127xDriver::TXnbISR()
 
 void ICACHE_RAM_ATTR SX127xDriver::TXnb(uint8_t volatile *data, uint8_t length)
 {
-  if (instance->currOpmode == SX127x_OPMODE_TX)
-  {
-    Serial.println("abort TX");
-    return; // we were already TXing so abort
-  }
+  // if (instance->currOpmode == SX127x_OPMODE_TX)
```

Comment:
Do we still want this code? Should it still be commented out?

Reply:
I'd leave to leave it in for debug. It'll trigger if the TX windows overlap